### PR TITLE
polished the readme.md and index.js 

### PR DESCRIPTION
**Pull request by Whirl21**
i polished the readme.md and index.js and added a file for djs example
chnaging the values in readme.md is easy....

### CODE REVIEW

File: .replit [hunk 1]
```diff
@@ -0,0 +1,2 @@
+language = "nodejs"
+run = "npm start"
```
Comment (dannyhpy) [inline .replit:2]: I'm not sure we need this file

File: examples/test.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const BrawlStars = require('..')
+const BrawlStars = require('brawlstars.js')
```
Comment (dannyhpy) [inline examples/test.js:1]: This was intentional, running this would result in an error since the module **brawlstars.js** cannot be resolved

Comment (Whirl21) [reply examples/test.js:1]: oh alright i'll make another

File: .replit (hunk 1 above)
Comment (Whirl21) [reply .replit:2]: uh oh that's replit 
well ig you can remove it..

File: examples/test.js (hunk 2 above)
Comment (dannyhpy) [reply examples/test.js:1]: You mean, an another PR? You can make changes in this one, I don't mind

**Pull request closed without merging**